pkgs/top-level/stage.nix

```diff
@@ -241,6 +246,7 @@ let
     platformCompat
     stdenvAdapters
     trivialBuilders
+    testUtilities
```

Comment:
Do we really need another layer for this?
It seems like this could be added to `trivialBuilders` instead.

Reply:
That's definitely possible, but `trivialBuilders` already exports about 15 different utilities. Putting test utilities in their own file seemed more readable to me.

Perhaps that was premature, and we should revisit thinking about splitting that file when we have some more test utilities (or find another axis to split `trivialBuilders` on). I'll put it in `trivialBuilders` for now.